kommunicateui/src/main/java/com/applozic/mobicomkit/uiwidgets/conversation/fragment/MobiComConversationFragment.java

```diff
@@ -1232,9 +1241,6 @@ protected void sendMessage() {
                 processSendMessage();
             }
         }
-        if (contact != null && !contact.isBlocked() || channel != null) {
-            handleSendAndRecordButtonView(false);
-        }
     }
```

Comment:
removing this from here won't affect normal flow ?

Reply:
No, because we have added handleSendAndRecordButtonView were already present in other places and I have tested and added handleSendAndRecordButtonView wherever it was required.